crates/ruff_python_formatter/Cargo.toml

```diff
@@ -38,7 +38,7 @@ tracing = { workspace = true }
 unicode-width = { workspace = true }
 
 [dev-dependencies]
-ruff_formatter = { path = "../ruff_formatter", features = ["serde"] }
+ruff_formatter = { path = "../ruff_formatter" }
```

Comment:
What’s this?

Reply:
A fix-up from my changes. Could have been it's own PR too, I guess. There's a `serde` feature. The `ruff_formatter` serde feature should depend on whether `serde` is enabled for `ruff_python_formatter`